pkg/kubectl/cmd/set/set_resources.go

```diff
@@ -87,17 +85,11 @@ func NewCmdResources(f cmdutil.Factory, out io.Writer, errOut io.Writer) *cobra.
 		Out: out,
 		Err: errOut,
 	}
-	var pod_specs string
-	RESTMappings := cmdutil.ResourcesWithPodSpecs()
-	for _, Map := range RESTMappings {
-		pod_specs = pod_specs + ", " + Map.Resource
-
-	}
 
 	cmd := &cobra.Command{
 		Use:     "resources (-f FILENAME | TYPE NAME)  ([--limits=LIMITS & --requests=REQUESTS]",
 		Short:   "update resource requests/limits on objects with pod templates",
-		Long:    resources_long + " " + pod_specs[2:],
+		Long:    resources_long,
```

Comment:
You can probably preserve this by adding a method onto Factory to return the equivalent, and just use that.  It needs to be a fast call, so I would think it would be something like

```
type Factory interface {
  ...
  SuggestPodTemplateResources() []unversioned.GroupResource
}
```

That method MUST not call to the network, so everything has to be hardcoded.  The doc would say it must be fast and succinct (return X most important resources).